wpilib/wpilib/analogpotentiometer.py

```diff
@@ -60,7 +60,7 @@ def get(self):
         :returns: The current position of the potentiometer.
         :rtype: float
         """
-        return (self.analog_input.getVoltage() / hal.getUserVoltage5V()) * self.fullRange + self.offset
+        return (self.analog_input.getVoltage() / hal.getUserVoltage5V()) * self.fullRange + self._offset
```

Comment:
Somehow I missed this (perhaps because of fullrange), but line 53 says that offset doesn't start with an underscore.